src/css/cs/items/_aa_planner.scss

```diff
@@ -9,6 +9,7 @@
         font-size: 20px;
         margin: 10px;
         max-width: 30%;
+        min-width: 30%;
```

Comment:
@darrylumich it looks like you've changed your master branch and not a SISRP branch.
Can you make sure you make the PR from a SISRP branch?

And instead of adding `min-width`, we should change `max-width` to `width`.
So in the end you should have `width: 30%`.

The way to do this is the following:
1. Make the change to the file (remove `min-width`, change 'max-width`)
2. git add -A
3. git commit -a
4. git push origin SISRP-XXXX
5. Make a new PR